packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2437,13 +2437,13 @@ class RenderRepaintBoundary extends RenderProxyBox {
         } else {
           diagnosis = 'this repaint boundary is not very effective and should probably be removed';
         }
-        description.add('metrics: ${percentage.toStringAsFixed(1)}% useful ($debugSymmetricPaintCount bad vs $debugAsymmetricPaintCount good)');
-        description.add('diagnosis: $diagnosis');
+        description.add(new StringProperty('metrics', '${percentage.toStringAsFixed(1)}% useful ($debugSymmetricPaintCount bad vs $debugAsymmetricPaintCount good)'));
+        description.add(new StringProperty('diagnosis', diagnosis));
```

Comment:
Changed 
metrics to be a PercentProperty

Added an optional name argument to message so it can be used for the second case.
'name' argument to the message constructor so that this case can still be handled by 
message